app/controllers/folders_controller.rb

```diff
@@ -7,6 +7,10 @@ def index
 
   def show
     @folder = policy_scope(Folder).find(id)
+    search_results ||= FolderSignService.new(search:, relation: policy_scope(Sign), folder: @folder).process
```

Comment:
nit: there's no point conditionally assigning here

```suggestion
    search_results = FolderSignService.new(search:, relation: policy_scope(Sign), folder: @folder).process
```